src/utils/utils.py

```diff
@@ -31,6 +32,8 @@
 
 TOKENIZER = MosesTokenizer()
 SOS_TOK, EOS_TOK = "<SOS>", "<EOS>"
+BERT_TOKENIZER = None # NOTE(Alex): hacky way of not loading the tokenizer every time
```

Comment:
This seems more than a bit error-prone. Can we just have a `get_tokenizer(name)` function and memoize it? Something like https://docs.python.org/3.6/library/functools.html#functools.lru_cache should work